systatus.go

```diff
@@ -27,7 +28,12 @@ type EnvResponse struct {
 }
 
 func Enable(opts SystatusOptions) {
-	http.HandleFunc(fmt.Sprintf("%s/health", opts.Prefix), handleHealth)
+	if opts.Healthcheck == nil {
+		http.HandleFunc(fmt.Sprintf("%s/health", opts.Prefix), handleHealth)
+	} else {
+		http.HandleFunc(fmt.Sprintf("%s/health", opts.Prefix), opts.Healthcheck)
+	}
```

Comment:
My intention was to run a healthcheck function inside the handler, not directly override the handler

Do you think this is a feasible thing to do?

Reply:
Hi @Arghya721 never mind, I think we can go with this implementation and then refine it later.

Reply:
Yes, this is feasible, we can directly pass this custom health check function into the handler.
```
type SystatusOptions struct {
	Prefix      string
	ExposeEnv   bool
	Healthcheck func() error
}

...
http.HandleFunc(fmt.Sprintf("%s/health", opts.Prefix), func(w http.ResponseWriter, r *http.Request) {
		handleHealth(w, r, opts.Healthcheck)
})
```
Then we can check if the user has passed this custom function or not by the similar method used above.